scripts/ebcli_installer.py

```diff
@@ -151,6 +151,31 @@ def _exec_cmd(args):
 """
 
 
+INSTALLATION_SUCCESS_MESSAGE_WITH_EXPORT_RECOMMENDATION__NON_WINDOWS__PYTHON_INCLUDED = """
```

Comment:
We discussed about not touching this file at all.

Reply:
Ah looks like that was just an artifact from the previous commit, I'll remove it now

Reply:
Updated that file